build-bot-status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import sys
+import time
+import threading
+
+import requests
+
+from light_controller import LightController
+
+
+class BuildBotSession:
+    failColour = "red"
+    busyColour = "orange"
+    goodColour = "green"
+    exceptionColour = "purple"
+
+    def __init__(self, builderURL, slaveName):
```

Comment:
here should be BuildStatusDAO and LightController as input
We should create new class for BuildStatusDAO with input of builderURL and slave name. In order to extend with different stage dev, gamma, prod we can simply put them in a config file in format like
dev.url = 
dev.slaveName = 

gamma.url
gamma.slaveName

prod.url
prod.slaveName